make/scripts/compare.sh

```diff
@@ -319,15 +292,10 @@ compare_general_files() {
     OTHER_DIR=$2
     WORK_DIR=$3
 
-    GENERAL_FILES=$(cd $THIS_DIR && $FIND . -type f ! -name "*.so" ! -name "*.jar" \
-        ! -name "*.zip" ! -name "*.debuginfo" ! -name "*.dylib" ! -name "jexec" \
-        ! -name "modules" ! -name "ct.sym" ! -name "*.diz" ! -name "*.dll" \
-        ! -name "*.cpl" ! -name "*.pdb" ! -name "*.exp" ! -name "*.ilk" \
-        ! -name "*.lib" ! -name "*.jmod" ! -name "*.exe" \
-        ! -name "*.obj" ! -name "*.o" ! -name "jspawnhelper" ! -name "*.a" \
-        ! -name "*.tar.gz" ! -name "gtestLauncher" \
-        ! -name "*.map" \
-        | $GREP -v "./bin/"  | $SORT | $FILTER)
+    GENERAL_FILES=$(cd $THIS_DIR && $FIND . -type f ! -name "*.pdb" \
+        | $GREP -v "/hotspot/gtest/server/gtestLauncher" \
+        | $GREP -v "/hotspot/gtest/server/libjvm.dylib" \
+        | $SORT | $FILTER)
```

Comment:
This list of exclusion, or at least most of it, isn't for handling exceptions but because those files are handled explicitly by other comparison functions.

Reply:
I see. I'll split it into two parts then, one that deselects files that are guaranteed to be processed elsewhere, and one to filter out problematic files (currently only the gtest binaries and *.pdb).